README.md

```diff
@@ -69,30 +69,41 @@ To start a new `Span`, you can use the `StartSpanFromContext` method.
 
 ### Starting an empty trace by creating a "root span"
 
-It's always possible to create a "root" `Span` with no parent or other causal
-reference.
+It's always possible to create a "root" `Span` with no parent or other causal reference.
 
 ```php
-    $span = $tracer->startSpan('my_span');
+    $span = $tracer->startActiveSpan('my_first_span');
     ...
     $span->finish();
 ```
 
-### Creating a (child) Span given an existing (parent) Span
+#### Creating a child span assigning parent manually
 
 ```php
-    use OpenTracing\SpanReference\ChildOf;
+	use OpenTracing\SpanReference\ChildOf;
+	
+	$parent = GlobalTracer::getGlobalTracer()->startManualSpan('parent');	$child = GlobalTracer::getGlobalTracer()->startManualSpan('child', [
+		'child_of' => $parent
+	]);
+	...
+	$child->finish();
+	...
+	$parent->finish();
+```
 
-    function xyz(Span $parentSpan, ...) {
-        ...
-        $span = GlobalTracer::getGlobalTracer()->startSpan(
-            'my_span',
-            ChildOf::withContext($span->context())
-        );
-        
-        $span->finish();
-        ...
-    }
+#### Creating a child span using automatic active span management
+Every new span will take the active span as parent and it will take its spot.
+
+```php    
+	$parent = GlobalTracer::getGlobalTracer()->startManualSpan('parent');        ...
+
+    // Since the parent span has been created by using startActiveSpan we don't need
```

Comment:
You use `startManualSpan` for the parent here. I think this is a copy-and-paste mistake, since you want to demonstrate the use of `startActiveSpaǹ`.